### PR TITLE
Fix the Heat Pump Water Heater Information table

### LINKED ISSUE #10213: HPWH Information table is messed up
Issue overview
--------------
image

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by shorowit**
Pull request overview
---------------------
 - Fixes #10213

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [x] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [x] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by shorowit** [comment]
Using the WaterHeaterHeatPumpWrappedCondenser.idf test file.

Before:

image

After:

(Updated; see below)

**Comment by shorowit** [comment]
Technically an `OutputChange`.

**Comment by mjwitte** [comment]
> Technically an `OutputChange`.

Does the column header need the "(HPWH Qualification)"? The table is only for HWPH anyway, right?

**Comment by shorowit** [comment]
> Does the column header need the "(HPWH Qualification)"? The table is only for HWPH anyway, right?

Good point. I was trying to preserve the intent of the original code, but I agree with you. I'll make that change.

**Comment by shorowit** [comment]
Using the WaterHeaterHeatPumpWrappedCondenser.idf test file.

Before:

image

After:

image

**Comment by mjwitte** [comment]
@Myoldmopar @jmarrec Ok to let this in after I/O Freeze?

And if yes, @shorowit, it will need an entry in the OutputChanges doc.

**Comment by shorowit** [comment]
Yep, happy to make the change to the OutputChanges doc, just wasn't sure if it will be in the `23-1-0-to-23-2-0` file or a new `23-2-0-to-24-1-0` file.

Note that https://github.com/NREL/EnergyPlus/pull/10209 is a similar situation. Sorry I caught these so late in the release cycle.

**Comment by jmarrec** [comment]
Is this an existing issue with 23.1.0 or something new?
Regardless, iofreeze shouldn't mean we let broken things fly by, I support merging it anyways, but that's only my two cents.

**Comment by shorowit** [comment]
Existing issue with 23.1.0.

File: src/EnergyPlus/WaterThermalTanks.cc [hunk 1]
```diff
@@ -4183,7 +4183,7 @@ bool GetWaterThermalTankInput(EnergyPlusData &state)
                 "Standard Rated Energy Factor\n");
             static constexpr std::string_view Format_721(
                 "! <Heat Pump Water Heater Information>,Type,Name,Volume {{m3}},Maximum Capacity {{W}},Standard Rated Recovery "
-                "Efficiency,Standard Rated Energy Factor,\"DX Coil Total Cooling Rate {{W, HPWH Only}}\"\n");
+                "Efficiency,Standard Rated Energy Factor,DX Coil Total Cooling Rate {{W}}\n");
```
Comment (rraustad) [inline src/EnergyPlus/WaterThermalTanks.cc:4186]: It appears this was intended to be a single column. This change seems reasonable.

File: src/Transition/OutputRulesFiles/OutputChanges23-1-0-to-23-2-0.md [hunk 2]
```diff
@@ -69,3 +69,9 @@ See pull request [#9982](https://github.com/NREL/EnergyPlus/pull/9982) for more
  (d) "Steam" to "DistrictHeatingSteam"
 
  See [9260](https://github.com/NREL/EnergyPlus/pull/9260)
+
+### Heat Pump Water Heater Information table
+
+Columns `"DX Coil Total Cooling Rate {W` and `HPWH Only}"` have been merged into a single `DX Coil Total Cooling Rate {W}`.
+
+ See [10214](https://github.com/NREL/EnergyPlus/pull/10214)
```
Comment (rraustad) [inline src/Transition/OutputRulesFiles/OutputChanges23-1-0-to-23-2-0.md:77]: All requirements for this type of change have been addressed.

**Review by rraustad — APPROVED** [review]
Looks good.

**Comment by rraustad** [comment]
Pulled develop, unit tests run successfully.

HeatPumpWaterHeater with Denver Intl Ap,CO.

Develop:

image

This branch:

image

@Myoldmopar this branch is ready to merge.

**Comment by Myoldmopar** [comment]
Thanks @rraustad I'll just do a quick build since several PRs have merged.

**Comment by Myoldmopar** [comment]
Yep, all happy, merging.  Thanks @shorowit and @rraustad

**Pull request merged**